src/app/course/course-question-snippet/course-question-snippet.component.ts

```diff
@@ -169,9 +169,30 @@ export class CourseQuestionSnippetComponent implements OnInit {
             }).subscribe()
         } else {
             this.router.navigate(
-                ['../' , this.eventId, 'problem', uqj.question.id, 'edit']
-                ,{relativeTo: this.route}
+                ['../', this.eventId, 'problem', uqj.question.id, 'edit']
+                , {relativeTo: this.route}
             ).then()
         }
     }
+
+    /**
+     * Opens the dialog service based on the template passed
+     * @param content - the template to be used
+     * @param eventRemove - the event to be removed
+     */
+    openRemoveQuestionInEventDialog(
+        content: PolymorpheusContent<TuiDialogContext>,
+        eventRemove: CourseEvent
+    ): void {
+        let labelText
+        if (eventRemove.is_closed && !eventRemove.is_not_available_yet) {
+            labelText = 'Remove question in finished assessment?'
+        } else {
+            labelText = 'Remove question?'
+        }
```

Comment:
I think `is_closed` means that the event has finished so you don't need the other clause. (Double check this please!!!)
```suggestion
        let labelText = 'Remove question?'
        if (eventRemove.is_closed) {
            labelText = 'Remove question in finished assessment?'
        }
```

Reply:
you were right! Whoops, I must have gotten confused by a previous conversation when discussing that popups shouldn't happen for not yet available. I have added a task to small changes to go back through all the old code and apply this fix.